### DISCUSSION TITLE
New Profile Achievements

### DISCUSSION THREAD

**Discussion by byronbytes** [community/community#18161] [Other Feature Feedback, Questions, & Ideas] [answered]
There's a new feature on my profile that shows achievements such as YOLO and Pull Shark.

 In my opinion, as said by others the designs could be more professional and the shapes are a little out of shape compared to the others, but since it's in beta I understand. Is there a list of all available new achievements? 

Maybe also a way to toggle them would be nice as well.

**Comment by uhKieran** [comment, marked as answer]
You can toggle the achievements on your profile in the settings page.
A list of all available new achievements can be found here: https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/ :)

**Reply by byronbytes** [reply]
Oh, thank you! I'm glad that there's going to be more planned in the future because this feature _could_ go really well if executed correctly.

**Reply by uhKieran** [reply]
You're welcome! :)

**Reply by yaya2devops** [reply]
Very cool stuff from Github!

**Reply by ehnruiz** [reply]
Awesome way to get the community engaged! Thanks 👍

**Reply by manuelabarca** [reply]
Thank for list

**Reply by Kasunm12** [reply]
ok.Get the community engaged! thanks

**Reply by matcldr** [reply]
I love the designs 😍

**Comment by Falci** [comment]
I hope my commit pushes direct to main branch also counts as YOLO

**Comment by Andyveloper** [comment]
I like the idea but I agree with you that they should look a bit more professional and maybe should be smaller as well

**Reply by matcldr** [reply]
I agree with you they should look a bit smaller

**Comment by Synceratus** [comment]
Awesome feature, ngl

**Comment by matcldr** [comment]
Achievements look cute ngl

**Comment by cbendot** [comment]
cool and tanks for add this

**Comment by dacbd** [comment]
all I want is a cute [10k badge](https://github.blog/2021-10-21-github-marketplace-welcomes-its-10000th-action/) ❤️

**Comment by rafael3do** [comment]
very good

**Comment by Terrorboy** [comment]
I agree. It would be nice to have individual toggles

**Reply by byronbytes** [reply]
Yeah, I'm hoping for this as well

**Comment by devdeem** [comment]
Achievements look good ngl
